msbuild/Xamarin.MacDev.Tasks/Xamarin.MacDev.Tasks.csproj

```diff
@@ -41,6 +41,7 @@
     <Reference Include="System.Xml" />
     <Reference Include="Mono.Cecil, Version=0.9.5.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756">
```

Comment:
@migueldeicaza here you can see it's not being said from where to get Cecil, which is why just opening the solution and trying to build on VS fails miserably out of the box ;)